src/client/node-client.js

```diff
@@ -213,19 +213,21 @@ class Client {
    * do not update channel state so they will return a result immediately
    * without hitting the consensus pool.
    *
-   * @param call JavaScript dict representing the XDR call, function name and
-   *             args to be sent to the read-only node.
+   * @param fnt Function to be invoked.
+   * @param params Rest parameter representing the parameters to pass to the function.
    *
    * @return Promise that on success provides an XDR ReadonlyResponse.
   */
-  async readonlySubmit (call) {
+  async readonlySubmit (fnt, ...params) {
```

Comment:
I think this function should just take params as an argument, not using the Rest parameter:
`readonlySubmit (fnt, params)`

Tested an instance of the Contract Client with these changes and got an error doing the fromJSON on the call object.

Reply:
I think you're referring to this line. https://github.com/kochavalabs/mazzaroth-js/blob/develop/src/client/contract-client.js#L219 Would you prefer to expand the list here or just work with a simple list?

Reply:
Oh yeah, you are right.  That is where this is breaking.  If you expand the list on the contract-client call I think this would be good.  Then we can still call this function with no parameters like in the test.

Reply:
Done. thanks.